src/objdictgen/node.py

```diff
@@ -53,7 +54,7 @@
 Fore = colorama.Fore
 Style = colorama.Style
 
-RE_NAME = re.compile(r'(.*)\[(.*)\]')
+RE_NAME = re.compile(r'(.*)\[[(](.*)[)]\]')
```

Comment:
Why this change? Please add a comment for this. (But in another PR, please).

Reply:
This regex expression is used to match variable names from maps.py. It is only used in the StringFormat(..) method and is used to separate the unformatted string from the parameters to be used in the formatting. When I removed the use of eval() I adjusted the regex to match and find the parameters within the brackets, rather that including the round brackets. I then treat this as a comma separated list rather than using it directly in the string formatting with the eval() method.

This way of formatting the names in now partially baked into the .jsonc implementation. See an example from a new OD below:

`"index": "0x1A00",  // 6656
      "name": "Transmit PDO %d Mapping[(idx)]",
      "struct": "narray",
      "group": "built-in",
      "mandatory": false,
      "incr": 1,
      "nbmax": 512,
      "each": {
        "name": "PDO %d Mapping for a process data variable %d[(idx,sub)]",
        "type": "UNSIGNED32",  // 7
        "access": "rw",
        "pdo": false,
        "nbmin": 0,
        "nbmax": 64
      },`

Reply:
Change has now been removed from this PR.